frontend/src/components/Topbar.jsx

```
@@ -69,9 +71,25 @@ export const Topbar = () => {
+            }
+          }}
+        >
+          STUDIFY
+        </div>
+
         <div className={styles.rightSide}>
-          <button className={styles.notification} onClick={handleNotificatonBtn}>
+          <button
+            className={styles.notification}
+            onClick={handleNotificatonBtn}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)